pkg/sqlite/load.go

```diff
@@ -53,9 +53,7 @@ func NewSQLLiteLoader(outFilename string) (*SQLLoader, error) {
 		replaces INTEGER,
 		depth INTEGER,
 		FOREIGN KEY(replaces) REFERENCES channel_entry(entry_id)  DEFERRABLE INITIALLY DEFERRED, 
-		FOREIGN KEY(channel_name) REFERENCES channel(name),
-		FOREIGN KEY(package_name) REFERENCES channel(package_name),
-		FOREIGN KEY(operatorbundle_name) REFERENCES operatorbundle(name)
```

Comment:
Had to remove the foreign key to operator_bundle, since synthesized channel entries for `skip`ed releases may not have a corresponding bundle.